pyomo/contrib/mindtpy/tests/MINLP2_simple.py

```diff
@@ -4,7 +4,7 @@
 MINLP test problem in Pyomo
 Author: David Bernal <https://github.com/bernalde>
 
-The expected optimal solution value is 6.0072.
+The expected optimal solution value is 6.00976.
```

Comment:
Huh? Was this simply wrong before?

Reply:
Yeah it's literally different in the paper.
See https://www.sciencedirect.com/science/article/pii/009813549587027X Table 1 (page 5)

Reply:
The paper also says that Duran and Grossmann (1986) give a solution of 6.00972. Looks like numerical noise to me, so I suppose it's fine. What solution do you end up getting?

Reply:
You're right, both previous versions had a minor mistake.
Fixed in aafd64294b77d34eebc452fd2c4b56e023aadddc